lib/resources/kernel_module.rb

```diff
@@ -37,7 +38,7 @@ def loaded?
     end
 
     def version
-      if inspec.os[:name] == 'centos' && inspec.os[:release].to_i == 5
+      if inspec.os.family == 'redhat'
```

Comment:
Yes, If you don't want use `os.family`, we can use 'os.redhat? || os.name == 'fedora'` (I have found the problem on fedora too).